cinema/migrations/0003_alter_movie_description.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.4 on 2023-08-23 13:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0002_alter_movie_duration'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='description',
+            field=models.CharField(max_length=255, null=True),
+        ),
+    ]
```

Comment:
Add one migration with the `CreateModel` operation that encounters all necessary changes.